ci-operator/step-registry/baremetal/lab/agent/install/network/static/disconnected/day2/baremetal-lab-agent-install-network-static-disconnected-day2-workflow.yaml

```diff
@@ -5,7 +5,7 @@ workflow:
       - chain: baremetal-lab-agent-conf-disconnected-day2
       - ref: baremetal-lab-agent-conf-static
       - ref: baremetal-lab-agent-install
-      - ref: upi-install-heterogeneous
+      - ref: baremetal-lab-agent-day2
```

Comment:
I do not remember why we added this here, but it is part of the `tests` chain for other platforms as this is a day2 operation. Failures in the pre-chain are reported as failures for the installer team in RP.


Also, consider creating a junit and checking that the COs and nodes are ready and available before moving forward. 

In the past, some bugs escaped because of the missing failure report in RP (when using other architectures in particular).

Now, RP reports failures regarding the `test` chain in the `CI (Overall)` test suite, but I don't remember if we are reporting back the failures in the `pre` chain, too. 

@mhanss for other thoughts, too.